roles/example_cnf_deploy/scripts/get-example-cnf-results.sh

```
@@ -26,17 +26,29 @@ echo
 
-# Retrieve TestPMD results
-echo "--- Last statistics reported by TestPMD ---"
+# Retrieve CNFApp results
+echo "--- Last statistics reported by $CNFAPP_POD ---"
 echo
-$OC_BINARY logs -n "$APP_NAMESPACE" "$TESTPMD_POD" | tail -n22
+# Command for grout
+if [[ "$CNFAPP_POD" =~ "grout" ]]; then
+    INTERMEDIATE_FILE_NAME=grout_logs.txt
+    $OC_BINARY logs -n "$APP_NAMESPACE" "$CNFAPP_POD" | tail -n40 > "$INTERMEDIATE_FILE_NAME"
+    GROUT_LOG_LINES=($(cat -n "$INTERMEDIATE_FILE_NAME" | grep -F "*" | awk '{print $1}'))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)